locale/an/symbols.dic

```diff
@@ -1,22 +1,17 @@
-#Aragonese locale/ar/symbols.dic
+﻿# This file is generated automatically by the script import-symbols.py during make
```

Comment:
Thinking about it... I'd say do not put the details in the file itself, but in a README.md file in locale/ . In the files, just mentioned that they are generated automatically, and mention the README.md file.